### DISCUSSION TITLE
Is there a roadmap?

### DISCUSSION THREAD

**Discussion by ygoe** [bitfireAT/webdav-push#27] [General]
Hi,

Is there a roadmap for this feature? Right now it looks like a rough idea that isn't considered by anybody. Will this turn into an RFC standard related to the existing WebDAV standards? Will there be advertising to other apps to implement this? I'd be interested in some sequence of steps that have been done and that are still needed to bring this to reality. In its current form, I wouldn't consider this for new implementations and rather build my own simpler system.

**Comment by devvv4ever** [comment]
Hi! 

Our "unofficial" roadmap is:

* Making the Draft (most important parts are completed) which should turn into an RFC as an optimal result: https://github.com/bitfireAT/webdav-push/blob/main/webdav-push-draft.md
* Implementing this on the Nextcloud server (will be a plugin that users can simply activate) (partially done already - it is currently being worked on)
* Implement this in DAVx5 (first steps have been done already - and work on it will continue in the next weeks)
* After everything is working we will release this to production. And from there on we'll advertise it to other server vendors at least, because we have quite good connections across the all the services and servers providing the DAV* protocol. We'll reference to the live and working implementation that we do together with Nextcloud and encourage them to implement it, too, hoping that the whole open ecosystem will be improved at the end :)
* We hope to have a working push with DAVx5 and Nextcloud in late summer.

Thats our plan so far.
